lib/datadog/appsec/configuration.rb

```diff
@@ -47,6 +47,14 @@ def trace_rate_limit=(value)
           options[:trace_rate_limit] = value
         end
 
+        def obfuscator_key_regex=(value)
+          options[:obfuscator_key_regex] = value
+        end
+
+        def obfuscator_value_regex=(value)
+          options[:obfuscator_value_regex] = value
+        end
```

Comment:
Since the intent here is to make this user-configurable, may I suggest documenting what these do? (Or perhaps linking here to where we document this?)

Reply:
It's documented here: https://github.com/DataDog/documentation/blob/e43454555120bb3d584cb524c4c1ab8cd313e524/content/en/security_platform/application_security/setup_and_configure.md#data-security-considerations

... but I really don't like this documentation structure that covers all languages and tries to be synthetic at the same time. As is it's hard to get that Ruby specific configuration without making the docs more confusing.

I wish AppSec doc related to Ruby would live in and be sourced from this repo instead.

Reply:
I see. Can we leave a link to <https://docs.datadoghq.com/security_platform/application_security/setup_and_configure/#data-security-considerations> as a comment then? It's not great, but it's a start :)

Reply:
Will open a PR with doc updates.